terraform/aws/modules/efs/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "creation_token" {
+  type    = string
+  default = "test-efs"
+
+}
+variable "performance_mode" {
+  type    = string
+  default = "generalPurpose"
+
+}
+variable "throughput_mode" {
+  type    = string
+  default = "bursting"
+
+}
+variable "encryption_set" {
+  type    = bool
+  default = true
+}
+variable "kms_key_id" {
+  type    = string
+  default = "true"
+
+}
+variable "subnet_id" {
+  type    = string
+  default = "subnet-01234567890abcdef"
```

Comment:
pls change the default values, these and the sec groups provided below might not exist.